src/lib.rs

```diff
@@ -138,6 +138,30 @@ impl RenetChannelsExt for RepliconChannels {
     }
 }
 
+/// External trait for [`ClientId`] to provide convenient conversion into renet ClientId.
+pub trait RenetClientIdExt {
+    /// Returns server channel configs that can be used to create [`ConnectionConfig`](renet::ConnectionConfig).
```

Comment:
How about something like this?
```suggestion
    /// Returns renet's client ID.
```